templates/layout/base.html

```diff
@@ -24,14 +24,13 @@
             window.lazySizesConfig = window.lazySizesConfig || {};
             window.lazySizesConfig.loadMode = 1;
         </script>
-        <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js'}}"></script>
+        <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js' resourceHint='preload' as='script'}}"></script>
```

Comment:
this looks correct to me. `cdn` helper support optimal `as` attribute and `script` is one of its valid values.